backend/Dockerfile

```diff
@@ -7,10 +7,10 @@ RUN apt-get update && apt-get install -y unzip
 
 # Install Python dependencies
 COPY requirements.txt .
-RUN pip install -r requirements.txt
+RUN pip install --no-cache-dir -r requirements.txt
 
 # Copy application code
 COPY ./app /app
 
 # Run the FastAPI app
-CMD ["uvicorn", "main:app", "--host", "0.0.0.0", "--port", "8000", "--reload"]
+CMD ["uvicorn", "main:app", "--host", "0.0.0.0", "--port", "8000", "--proxy-headers"]
```

Comment:
### production

🌀  [No reloading] Implement something compatible for a production environment similar to other productions implementations such as this one:
https://github.com/SlideSpeak/pdf-to-docx-server/blob/81777f46a663226be1370e5e5ccbc838f7dcdc22/fastapi_server/Dockerfile#L17